workflow-templates/ci.properties.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "Cheqd Organization Workflow",
+    "description": "Cheqd Organization CI workflow template.",
```

Comment:
Perhaps rename to "Markdown quality control" as the name and description as "Markdown linting and link checking"

Reply:
Ok yes!